docs/config.md

```diff
@@ -220,16 +224,18 @@ model_names:
 
 ## `<objective>`
 
-Objectives allow you to specify the sorting criteria for the final results.
+Objectives specify the sorting criteria for the final results.
 The fields below are supported under this object type:
 
-1. `perf_throughput`
-2. `perf_latency`
-3. `gpu_used_memory`
-4. `gpu_free_memory`
-5. `gpu_utilization`
-6. `cpu_used_ram`
-7. `cpu_available_ram`
+| Option Name       |   Units   | Description                                            |
+| :---------------- | :-------: | :----------------------------------------------------- |
+| `perf_throughput` | inf / sec | Use throughput as the objective.                       |
```

Comment:
Why are units here? The value they put here is not the same as weights below and can cause confusion. Use units only when the user needs to enter a value in those units.